pmm-app/src/react-plugins-deps/components/helpers/styling.ts

```diff
@@ -0,0 +1,16 @@
+class Styling {
+  static setPluginPanelClass(panelWrapperId = '#antd', newClass = 'custom-grafana-plugin') {
```

Comment:
looks like `addPluginPanelClass` fits better for this method. Set usually means replace old value with a new one.